dev.env

```diff
@@ -0,0 +1,7 @@
+PORT=3000
+DISPATCHERQUEUE=dispatcher
+# the IP address of the host system (when running on Linux)
+RABBITHOST=172.18.0.1
```

Comment:
is this always the case?

Reply:
This is only for the test setup

Reply:
in the deployment you would have another ip address. If running withing docker-compose, the ip address would be the internal name of the service